easybuild/easyconfigs/i/IntelDAAL/IntelDAAL-2019.4.007.eb

```diff
@@ -0,0 +1,48 @@
+##
+# This is a contribution from DeepThought HPC Service, Flinders University, Adelaide, Australia
+# Homepage: 	https://staff.flinders.edu.au/research/deep-thought
+#
+# Authors::	Robert Qiao <rob.qiao@flinders.edu.au>
+# License::	Apache v2.0
+#
+# Notes::
+##
+
+easyblock = "Binary"
+
+name = 'IntelDAAL'
+version = '2019.4.007'
+
+homepage = 'https://software.intel.com/en-us/daal'
+description = """
+Intel® Data Analytics Acceleration Library (Intel® DAAL) is the library of 
+Intel® architecture optimized building blocks covering all stages of data analytics:
+data acquisition from a data source, preprocessing, transformation, data mining, modeling,
+validation, and decision making.
+ """
+
+toolchain = SYSTEM
+
+source_urls = ['https://github.com/intel/daal/releases/download/%(version_major)s_u%(version_minor)s']
+sources = ['l_daal_oss_p_%(version)s.tgz']
+checksums = ['bc1e521678443ed885c99f8b257d8db31e400e47ffe00c9b3fb0e80f1ad39119']
+
+extract_sources = True
+
+daal_dir = "linux/daal"
+tbb_dir = "linux/tbb"
```

Comment:
@robqiao these local variable names should now be prefixed with, for instance, `local_`, to distinguish them from easyconfig parameters (see https://easybuild.readthedocs.io/en/latest/Easyconfig-files-local-variables.html)

Reply:
Thx @migueldiascosta 👍 Fixed.